src/dos/dos.cpp

```diff
@@ -47,6 +48,138 @@ void DOS_SetError(Bit16u code) {
 	dos.errorcode=code;
 }
 
+int countries_date_format_yyyymmdd[] = {
+        2,   // Canadian-French
```

Comment:
Suggest using an enum class to turn county magic numbers into self-documenting code.
Use them like: `COUNTRY:Candian_French`, `COUNTRY:Hungary`, ...
Then the comments disappear.

Reply:
Rewrote the country code as enum class as suggested. Done.